components/api_version/src/lib.rs

```diff
@@ -71,28 +71,47 @@ pub trait APIVersion: Clone + Copy + 'static + Send + Sync {
 }
 
 #[derive(Default, Clone, Copy)]
-pub struct APIV1;
+pub struct ApiV1;
 #[derive(Default, Clone, Copy)]
-pub struct APIV1TTL;
+pub struct ApiV1Ttl;
 #[derive(Default, Clone, Copy)]
-pub struct APIV2;
+pub struct ApiV2;
+
+#[macro_export]
+macro_rules! check_kv_format {
```

Comment:
Suggest naming as something like `test_kv_format_impl`, means "test KvFormat implementations". `check_kv_format` would be confused with [`check_api_version`](https://github.com/tikv/tikv/blob/8b509a7415e137b3719b1a947d11dfbf20e3bbb5/src/storage/mod.rs#L409).

Rest LGTM~